### DISCUSSION TITLE
404 error when attempting to access domain

### DISCUSSION THREAD

**Discussion by stamatim** [vercel/community#5631] [Help] [answered]
### Question

I have a domain registered with a third-party provider which I had transferred to Vercel a few years ago. Today, I logged in to go update some DNS records, but when I tried to click on the option from the Domains page (`https://vercel.com/<user>/~/domains`), I was shown a 404 page that said "You are logged in as <vercel_user_email>" and a button below that prompted me to "Sign in as a different user". 

Not sure what to do. This has never been an issue in the past.

**Comment by JoseGermanx** [comment]
Hello I have the same problem. 
![image](https://github.com/vercel/community/assets/471118/b0f48068-272c-4fb9-8424-3de55802789b)

**Comment by stamatim** [comment]
Related Discussions:

- https://github.com/orgs/vercel/discussions/5625

**Reply by stamatim** [reply, marked as answer]
@JoseGermanx Looks like this was resolved. See this comment for a manual workaround as well.

https://github.com/orgs/vercel/discussions/5625#discussioncomment-8261722
